src/gballoc.c

```diff
@@ -117,7 +118,7 @@ void* gballoc_malloc(size_t size)
                 head = allocation;
 
                 g_allocations++;
-                totalSize += size;
+                totalSize = safe_add_size_t(totalSize, size);
```

Comment:
This should be checked for size_t overflow.

Reply:
not needed. This is only for malloc stats update